lib/Support/ErrorHandling.cpp

```diff
@@ -85,7 +92,15 @@ static void klee_vfmessage(FILE *fp, const char *pfx, const char *msg,
                        /*bg=*/false);
   }
 
-  fdos << "KLEE: ";
+  if (LogCpuTime) {
+    sys::TimeValue now(0,0), user(0,0), sys(0,0);
+    sys::Process::GetTimeUsage(now,user,sys);
+    fdos << (user.seconds() + sys.seconds()) << ".";
+    fdos << (user.nanoseconds() + sys.nanoseconds());
+  }
+
+  fdos << " KLEE: ";
```

Comment:
To make it easier to grep the output it would probably be more helpful to keep the KLEE prefix at the beginning. E.g.

```
KLEE: [ <seconds>.<nanoseconds> ]: <message>
```

Unless you have a particular reason you want the time to appear first?